pkg/client/statistics.go

```diff
@@ -0,0 +1,127 @@
+package client
+
+import (
+	"time"
+)
+
+// QueueStats stores statistics related to the queue processing
+type QueueStats struct {
+	// enqueued is number of items that has been accepted for processing
+	enqueued uint64 `mapstructure:"enqueued"`
+	// Processed is number of items that has been successfully processed
+	processed uint64 `mapstructure:"processed"`
+	// dropped is number of items that has been dropped since they couldn't be processed
+	dropped uint64 `mapstructure:"dropped"`
+	// broken is number of items that has been damaged by queue, should be zero
+	broken uint64 `mapstructure:"broken"`
+	// processingTime is duration of the processing
```

Comment:
I assume that when this value is logged to stdout/stderr, the unit we use is seconds, right? 

It would also help if the comment clarified if all those values are absolute (counters) vs relative (gauge). 

With absolute, I mean counters which are storing the values since the collector start / reload / similar and the value will always be increasing or if it's relative value compared to some previous point in time (e.g. since the last time statistics were logged).

If so, it may make sense to clarify that here.

Reply:
Those numbers are absolute. I can update the comment in 2hrs.

Reply:
Thanks.

Yeah that would be great. In the mean time, I will go ahead and approve the PR.

Thanks.